plugin/hover.py

```diff
@@ -38,6 +38,14 @@ def on_hover(self, point, hover_zone):
 _test_contents = []  # type: List[str]
 
 
+class_for_severity = {
+    DiagnosticSeverity.Error: 'errors',
+    DiagnosticSeverity.Warning: 'warnings',
+    DiagnosticSeverity.Information: 'info',
+    DiagnosticSeverity.Hint: 'hints'
+}
+
+
```

Comment:
You could use [`diagnostic_severity_names`](https://github.com/tomv564/LSP/blob/25e63ba9c7df8b92d21bc282cdf0fe1910e268a1/plugin/diagnostics.py#L23) for determining the class name?

Reply:
Good suggestion! As I'd have to move that lookup and change all the CSS names to singular I've avoided it for now.